client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "openmsupply-client",
-  "version": "1.0.5",
+  "version": "1.0.0-alpha.6",
```

Comment:
I wonder if we can just comment that version is in server/cargo.toml (or is it required in package.json ?)